src/route/common.py

```diff
@@ -0,0 +1,16 @@
+from flask import Blueprint, jsonify, current_app
+from flask_restful import Api
+
+
+common_blueprint = Blueprint('common', __name__)
+common_blueprint_api = Api(common_blueprint)
+
+
+@common_blueprint.route('/routes')
+def list_routes():
+    output = []
+    for rule in current_app.url_map.iter_rules():
+        methods = ','.join(rule.methods)
+        line = "{:50s} {:20s}".format(str(rule), methods)
+        output.append(line)
+    return jsonify(routes=output)
```

Comment:
What is the reason for this function?

Reply:
This function lists all available project URLs and returns them in JSON format.

Reply:
It is obvious, I mean what is the reason for that? In which case it could be usable?